app/src/main/res/values-de/strings.xml

```diff
@@ -127,25 +128,25 @@
   <string name="external_multiple_apps_matched_exit">Privaten Modus verlassen?</string>
   <string name="download_snackbar_finished">%1$s abgeschlossen</string>
   <string name="download_snackbar_open">Öffnen</string>
-  <string name="download_dialog_warning"><![CDATA["Heruntergeladene Dateien werden beim Löschen der Chronik von %1$s "<b xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">nicht gel&#246;scht</b>.]]></string>
+  <string name="download_dialog_warning"><![CDATA[\"Heruntergeladene Dateien werden beim Löschen der Chronik von %1$s \"<b xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">nicht gel&amp;#246;scht</b>.]]></string>
   <string name="download_dialog_title">Datei herunterladen</string>
   <string name="download_dialog_action_cancel">Abbrechen</string>
   <string name="download_dialog_action_download">Herunterladen</string>
   <string name="errorpage_title">Seiten-Ladefehler</string>
   <string name="errorpage_refresh">Erneut versuchen</string>
   <string name="error_connectionfailure_title">Verbindung fehlgeschlagen</string>
-  <string name="error_connectionfailure_message"><![CDATA[<ul xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">"\n        "<li>Die Website k&#246;nnte vor&#252;bergehend nicht erreichbar sein, versuchen Sie es bitte sp&#228;ter nochmals.</li>"\n        "<li>"Wenn Sie auch keine andere Website aufrufen k&#246;nnen, &#252;berpr&#252;fen Sie bitte die Daten- oder WLAN-Verbindung. "</li>"\n      "</ul>]]></string>
+  <string name="error_connectionfailure_message"><![CDATA[<ul xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">"\"\\n        \""<li>Die Website k&amp;#246;nnte vor&amp;#252;bergehend nicht erreichbar sein, versuchen Sie es bitte sp&amp;#228;ter nochmals.</li>"\"\\n        \""<li>\"Wenn Sie auch keine andere Website aufrufen k&amp;#246;nnen, &amp;#252;berpr&amp;#252;fen Sie bitte die Daten- oder WLAN-Verbindung. \"</li>"\"\\n      \""</ul>]]></string>
```

Comment:
Just want to double check that `\""` is expected?

Reply:
Talked to @Delphine about this - she said there's a problem with the pontoon scripts, but I think it's okay to merge these changes in. We'll be able to 1) see how the changes look in the screenshots, and 2) any new changes would be in new commits anyway.